packages/react-server/src/ReactFlightServer.js

```diff
@@ -4302,15 +4482,15 @@ function renderDebugModel(
     if (value.length >= 1024) {
       // Large strings are counted towards the object limit.
       if (counter.objectLimit <= 0) {
-        // We've reached our max number of objects to serialize across the wire so we serialize this
+        // We've reached our max number of objects to serializeDebug across the wire so we serializeDebug this
```

Comment:
Search & replace mistake?
```suggestion
        // We've reached our max number of objects to serialize across the wire so we serialize this
```